build/networking/README_DOCKER.md

```diff
@@ -193,67 +198,67 @@ below.
 
-```
-ipdk createvms
-```
+    ```bash
+    ipdk createvms
+    ```
 
-3) Those two created VM images can at any moment be started with:
+1. Those two created VM images can at any moment be started with:
 
-```
-ipdk startvms
-```
+    ```bash
+    ipdk startvms
+    ```
 
-*NOTE*: If VM's are not up even after waiting for 6-9 minutes, check if
-hugepages are mounted to `/mnt/huge`.
-  Example: Command to mount huge pages is `mount -t hugetlbfs nodev /mnt/huge`
+    *NOTE*: If VM's are not up even after waiting for 6-9 minutes, check if
+    hugepages are mounted to `/mnt/huge`.
+      Example: Command to mount huge pages is `mount -t hugetlbfs nodev /mnt/huge`
 
-4) Create a forwarding pipeline program by compiling and package the vSwitch consumable
+1. Create a forwarding pipeline program by compiling and package the vSwitch consumable
 pipeline binary package by using the pipeline builder:
 
-```
-export OUTPUT_DIR=/root/examples/simple_l3
-ipdk execute --- p4c --arch psa --target dpdk --output $OUTPUT_DIR/pipe --p4runtime-files $OUTPUT_DIR/p4Info.txt --bf-rt-schema $OUTPUT_DIR/bf-rt.json --context $OUTPUT_DIR/pipe/context.json $OUTPUT_DIR/simple_l3.p4
+    ```bash
+    export OUTPUT_DIR=/root/examples/simple_l3
+    ipdk execute --- p4c --arch psa --target dpdk --output $OUTPUT_DIR/pipe --p4runtime-files $OUTPUT_DIR/p4Info.txt --bf-rt-schema $OUTPUT_DIR/bf-rt.json --context $OUTPUT_DIR/pipe/context.json $OUTPUT_DIR/simple_l3.p4
 
-ipdk execute /root/examples/simple_l3 --- tdi_pipeline_builder --p4c_conf_file=simple_l3.conf --bf_pipeline_config_binary_file=simple_l3.pb.bin
-```
+    ipdk execute /root/examples/simple_l3 --- tdi_pipeline_builder --p4c_conf_file=simple_l3.conf --bf_pipeline_config_binary_file=simple_l3.pb.bin
+    ```
 
-5) Add the created pipeline binary package to the running IPDK container infrap4d switch:
+1. Add the created pipeline binary package to the running IPDK container infrap4d switch:
 
-```
-ipdk execute --- p4rt-ctl set-pipe br0 /root/examples/simple_l3/simple_l3.pb.bin /root/examples/simple_l3/p4Info.txt
-```
+    ```bash
+    ipdk execute --- p4rt-ctl set-pipe br0 /root/examples/simple_l3/simple_l3.pb.bin /root/examples/simple_l3/p4Info.txt
+    ```
 
-6) Add pipeline table rules:
+1. Add pipeline table rules:
 
-```
-ipdk execute --- p4rt-ctl add-entry br0 ingress.ipv4_host "hdr.ipv4.dst_addr=1.1.1.1,action=ingress.send(0)"
+    ```bash
+    ipdk execute --- p4rt-ctl add-entry br0 ingress.ipv4_host "hdr.ipv4.dst_addr=1.1.1.1,action=ingress.send(0)"
 
-ipdk execute --- p4rt-ctl add-entry br0 ingress.ipv4_host "hdr.ipv4.dst_addr=2.2.2.2,action=ingress.send(1)"
-```
+    ipdk execute --- p4rt-ctl add-entry br0 ingress.ipv4_host "hdr.ipv4.dst_addr=2.2.2.2,action=ingress.send(1)"
+    ```
 
 From this step you can connect to the test VMs and do some ping tests as described below!
 
 ### Connect to the test VMs serial consoles
 
 You will need two login windows, one for each VM:
```

Comment:
This use of _login_ is correct. It is an adjective, modifying the noun _windows_.